apps/cow-fi/components/TopicPageComponent.tsx

```diff
@@ -92,11 +93,12 @@ const CategoryDescription = styled.div`
 
 interface TopicPageProps {
   category: any
-  articles: any[]
+  articles: Article[]
   allCategories: { name: string; slug: string }[]
+  allArticles?: Article[]
```

Comment:
The only place where this component is in use is `TopicPage` and `allArticles` prop is always set there. It seems the prop cannot be optional.